docs/docs/docs/01-core/admin/01-ee/02-audit-logs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Audit Logs
+description: Guide for Audit Logs
+tags:
+  - settings
+  - audit-logs
+---
+
+# Audit Logs
+
+## Summary
+
+Audit Logs provide a way to view the history of all user actions at Admin API level. This includes actions related to entries (including publish actions), media (including its folders), users, login & logout of admin users, components, roles, and permissions, you can see the list of all the default events [here](https://github.com/strapi/strapi/blob/main/packages/core/admin/ee/server/services/audit-logs.js#L9).
```

Comment:
I think it would be a good idea to make this list dynamic, and allow other plugins or providers to update the audit log events.

Is this something that has been discussed?

Reply:
woops, missed the note! no prob. It's also something any team could implement in case it is needed :)